node/test/unit/tessel.js

```diff
@@ -1337,10 +1353,10 @@ exports['Tessel.Pin'] = {
       test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
```

Comment:
`0 << 3` is 0, and OR'ing with 0 doesn't do anything but truncate fractional bits (effectively, the operation is `ToInt32(...)`)

```js
// ASYNC_PIN_CHANGE_N starts at 0xC0
console.log((0xC0 === 0b11000000) && ((0b11000000 | 0) === 0b11000000)); // true
console.log((0xC1 === 0b11000001) && ((0b11000001 | 0) === 0b11000001)); // true
console.log((0xC2 === 0b11000010) && ((0b11000010 | 0) === 0b11000010)); // true
console.log((0xC3 === 0b11000011) && ((0b11000011 | 0) === 0b11000011)); // true
console.log((0xC4 === 0b11000100) && ((0b11000100 | 0) === 0b11000100)); // true
console.log((0xC5 === 0b11000101) && ((0b11000101 | 0) === 0b11000101)); // true
console.log((0xC6 === 0b11000110) && ((0b11000110 | 0) === 0b11000110)); // true
console.log((0xC7 === 0b11000111) && ((0b11000111 | 0) === 0b11000111)); // true

// ...

console.log((0b11000000 | 0) === (0b11000000 | (0 << 3))); // true
console.log((0b11000001 | 0) === (0b11000001 | (0 << 3))); // true
console.log((0b11000010 | 0) === (0b11000010 | (0 << 3))); // true
console.log((0b11000011 | 0) === (0b11000011 | (0 << 3))); // true
console.log((0b11000100 | 0) === (0b11000100 | (0 << 3))); // true
console.log((0b11000101 | 0) === (0b11000101 | (0 << 3))); // true
console.log((0b11000110 | 0) === (0b11000110 | (0 << 3))); // true
console.log((0b11000111 | 0) === (0b11000111 | (0 << 3))); // true
```

Which is just the illustrative way of saying that you can change these: `(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)` back to `REPLY.ASYNC_PIN_CHANGE_N + pinIndex` safely